release-notes/release-notes-v10.9.0.md

```diff
@@ -62,12 +62,14 @@
 - Use `satoshirate_normalized` and `give_quantity_normalized` to calculate `price_normalized`
 - Add a parameter `utxo_value` to the `attach.compose()` and `move.compose()` functions
 - Add `source_address` and `destination_address` in `sends` table
+- Use by default Blockstream API to get address UTXOs and history
```

Comment:
as fallback in bitcoin core doesn't have it?

Reply:
right! tweaked the sentence.